crates/api_models/src/payment_methods.rs

```diff
@@ -6,76 +6,153 @@ use utoipa::ToSchema;
 
 use crate::enums as api_enums;
 
-#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct CreatePaymentMethod {
+    /// The type of payment method use for the payment.
+    #[schema(value_type = PaymentMethodType,example = "card")]
     pub payment_method: api_enums::PaymentMethodType,
+    /// This is a sub-category of payment method.
+    #[schema(value_type = Option<PaymentMethodSubType>,example = "credit_card")]
     pub payment_method_type: Option<api_enums::PaymentMethodSubType>,
+    /// The name of the bank/ provider issuing the payment method to the end user
+    #[schema(example = "Citibank")]
     pub payment_method_issuer: Option<String>,
+    /// A standard code representing the issuer of payment method
+    #[schema(value_type = Option<PaymentMethodIssuerCode>,example = "JP_APPLEPAY")]
     pub payment_method_issuer_code: Option<api_enums::PaymentMethodIssuerCode>,
+    /// Card Details
+    #[schema(example = json!({
+    "card_number": "4111111145551142",
+    "card_exp_month": "10",
+    "card_exp_year": "25",
+    "card_holder_name": "John Doe"}))]
     pub card: Option<CardDetail>,
+    /// You can specify up to 50 keys, with key names up to 40 characters long and values up to 500 characters long. Metadata is useful for storing additional, structured information on an object.
+    #[schema(value_type = Option<Object>,example = json!({ "city": "NY", "unit": "245" }))]
     pub metadata: Option<serde_json::Value>,
+    /// the unique identifier of the customer.
+    #[schema(example = "cus_meowerunwiuwiwqw")]
     pub customer_id: Option<String>,
```

Comment:
Is the customer ID intentional? 🐈 

```suggestion
    /// The unique identifier of the customer.
    #[schema(example = "cus_meowerunwiuwiwqw")]
    pub customer_id: Option<String>,
```

Reply:
Hehe